### ISSUE TITLE
                                              54147 – Emphasize that a hyphen is Emphasize that a hyphen is really needed in "-1" by writing it as \(hy1

### ISSUE THREAD

**Issue by aelmahmoudy** [fribidi/fribidi#135]
While packaging fribidi, I got a lintian warning about fribidi_remove_bidi_marks.3 manpage, regarding the use of a dash (-) to indicate a hyphen in '-1', please use \(hy1 instead of -1

**Comment by dov** [comment]
I'm not to familiar with man page syntax, so could you please very that this this patch is ok:

```
modified   lib/fribidi.h
@@ -57,13 +57,13 @@
  * string, the other way. Moreover, the position maps should be filled with
  * valid entries.
  * 
- * A position map pointing to a removed character is filled with \-1. By the
+ * A position map pointing to a removed character is filled with \(hy1. By the
  * way, you should not use embedding_levels if str is visual string.
  * 
  * For best results this function should be run on a whole paragraph, not
  * lines; but feel free to do otherwise if you know what you are doing.
  *
- * Returns: New length of the string, or \-1 if an error occurred (memory
+ * Returns: New length of the string, or \(hy1 if an error occurred (memory
  * allocation failure most probably).
  */
 FRIBIDI_ENTRY FriBidiStrIndex
```

**Comment by aelmahmoudy** [comment]
Yes, that is correct

**Comment by dov** [comment]
👍

**Comment by dov** [comment]
But thinking about it a bit and doing a bit of Googling, why should a minus sign be written as a hyphen? The man page is not taking about `-foo` but about `-1`... Could you please give  a reference why `\(hy1` should be used?

**Comment by dov** [comment]
Reading up some more, shouldn't the correct usage be `\mi1`?

See e.g.: https://lists.libreplanet.org/archive/html/groff/2017-04/msg00052.html : 

>  - Educate people to always use \(mi rather than \- if they want a
>    typographic minus sign U+2212 MINUS SIGN (say, in mathematical
>    formula) as opposed to an ASCII minus sign (say, in sample C code).

**Comment by aelmahmoudy** [comment]
You might be right, I've filed this issue on fribidi's old bugzilla 
tracker years ago, and forgot about the details

-- 
‎أحمد المحمودي (Ahmed El-Mahmoudy)
 Digital design engineer
GPG KeyIDs: 4096R/A7EF5671 2048R/EDDDA1B7
GPG Fingerprints:
 6E2E E4BB 72E2 F417 D066  6ABF 7B30 B496 A7EF 5761
 8206 A196 2084 7E6D 0DF8  B176 BC19 6A94 EDDD A1B7

**Comment by dov** [comment]
Fixed in commit f503e57.
